src/service_client/assisted_service_api.py

```diff
@@ -161,9 +161,9 @@ def get_all_clusters(self) -> List[Dict[str, Any]]:
     def cluster_get(self, cluster_id: str) -> models.cluster.Cluster:
         return self.client.v2_get_cluster(cluster_id=cluster_id)
 
-    def get_infra_env_by_cluster_id(self, cluster_id: str) -> List[models.infra_env.InfraEnv]:
+    def get_infra_env_by_cluster_id(self, cluster_id: str) -> List[Union[models.infra_env.InfraEnv, Dict[str, Any]]]:
```

Comment:
which one is it?
it seems just ``List[Dict[str, Any]]`` no?

Reply:
Basically you right. But the attributes on this Dict are the properties of `models.infra_env.InfraEnv`. It's more for documentation and autocomplete